### ISSUE TITLE
Supports non visual reading: unknown

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#406]
In the principles document we have a string to indicate that the information on supports non-visual reading is unkown: "Not known if readable in read aloud and braille" (ID `nonvisual-reading-unknown`).

In the techniques, however, we do not have this situation because we indicate "May not be fully readable in read aloud and braille" (ID `nonvisual-reading-may-not-fully`) in the case of missing metadata.

**Comment by gregoriopellegrino** [comment]
We decided to change the techniques `nonvisual-reading-unknown`

**Comment by GeorgeKerscher** [comment]
This has been resolved.
